Formula/jabba.rb

```diff
@@ -22,14 +23,39 @@ class Jabba < Formula
   def install
     ENV["GOPATH"] = buildpath
     ENV["GO111MODULE"] = "auto"
+    ENV["JABBA_GET"] = "false"
+
+    inreplace "Makefile", " sh install.sh", " bash install.sh --skip-rc"
+    inreplace "install.sh" do |s|
+      # Call unaliased rm
+      s.gsub! "rm -f", "'rm' -f"
+
+      # Homebrew install locations
+      s.gsub! "$JABBA_HOME_TO_EXPORT/bin/jabba", "#{opt_bin}/jabba"
+      s.gsub! "${JABBA_HOME}/bin", bin.to_s
+      s.gsub! "${JABBA_HOME}/jabba.sh", "#{pkgshare}/jabba.sh"
+      s.gsub! "${JABBA_HOME}/jabba.fish", "#{pkgshare}/jabba.fish"
+    end
+
+    pkgshare.mkpath
+
     dir = buildpath/"src/github.com/Jabba-Team/jabba"
     dir.install buildpath.children
     cd dir do
```

Comment:
Does it work without this?

Reply:
It isn't, once we take out the `go` environment overrides.